tools/chocolateyInstall.ps1

```diff
@@ -1,7 +1,7 @@
 ﻿$packageName = "postman"
 $fileType = "exe"
 $silentArgs = "-s"
-$url = "https://dl.pstmn.io/download/latest/win32"
-$url64 = "https://dl.pstmn.io/download/latest/win64"
+$url = "https://dl.pstmn.io/download/version/6.7.3/windows3"
```

Comment:
Thanks for the contribution! However, this link doesn't work.

This would also be a good opportunity to add checksums to ensure file integrity (since this pull request targets the specific version). See https://chocolatey.org/docs/helpers-install-chocolatey-package. Or, I can add checksums if you prefer.

Reply:
This fixes if you'd like to use this suggestion:

```suggestion
$url = "https://dl.pstmn.io/download/version/6.7.3/windows32"
```